dotnet/src/Extensions/PromptTemplates.Liquid/LiquidPromptTemplate.cs

```diff
@@ -16,18 +16,24 @@ namespace Microsoft.SemanticKernel.PromptTemplates.Liquid;
 /// <summary>
 /// Represents a Liquid prompt template.
 /// </summary>
-internal sealed class LiquidPromptTemplate : IPromptTemplate
+internal sealed partial class LiquidPromptTemplate : IPromptTemplate
```

Comment:
@stephentoub can you clarify why the change to `partial` is needed here?

Reply:
required by the regex source generator `GeneratedRegex`

Reply:
https://youtu.be/ptKjWPC7pqw?t=1428